ros2topic/ros2topic/verb/info.py

```diff
@@ -29,7 +32,16 @@ def add_arguments(self, parser, cli_name):
 
     def main(self, *, args):
         with DirectNode(args) as node:
+            topic_names_and_types = get_topic_names_and_types(node=node)
             topic_name = args.topic_name
-            print('Topic: %s' % topic_name)
+            for (t_name, t_types) in topic_names_and_types:
+                if t_name == topic_name:
+                    topic_types = t_types
+                    break
+            else:
+                print("ERROR: Unknown topic '%s'" % topic_name, file=sys.stderr)
+                return
```

Comment:
Nitpick: this could be reduced to `return 'Unknown topic'` since the returned string is already printed to `stderr`. Also looking at the other command I would remove the `ERROR` prefix since none of the error message contains that.

Reply:
Thanks for the feedback will update it tomorrow.

The rationale for the error message format was to match the ROS 1 behavior. Will change to be more consistent with the rest of the code